include/transport/priority_queue.h

```diff
@@ -25,8 +25,8 @@ namespace qtransport {
     template<typename DataType, uint8_t PMAX = 32>
     class PriorityQueue
     {
-        using timeType = std::chrono::milliseconds;
-        using timeQueue = TimeQueue<DataType, timeType>;
+        using TimeType = std::chrono::milliseconds;
+        using TimeQueueUs = TimeQueue<DataType, TimeType>;
```

Comment:
I would instead recommend calling this `TimeQueue_t` or `TimeQueueType` since the time type (which is milliseconds not microseconds) isn't specific when it is passed to it. I.e. `TimeType` has no unit and can change to anything, so the queue type shouldn't have a unit associated with it either.